### ISSUE TITLE
Re-Burst feature request

### ISSUE THREAD

**Issue by ContemporaryInsanity** [wiqid/repelzen#9]
Not sure if issues is the right place for feature suggestions, apologies if not.

Would it be possible to have re-burst able to generate decelerating as well as accelerating bursts?

**Comment by wiqid** [comment]
yes, this is the right place. and it should be possible. i'll have a look.
